cmd/kops/util/validate_usage.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"fmt"
+	"k8s.io/kops/pkg/featureflag"
+	"k8s.io/kops/upup/pkg/fi"
+)
+
+const (
```

Comment:
this set of const seems unnecessary. I think it would be even easier to read if the unsupportedCommandsForVSphere  was simply a map of {"create -f": true} or even a list 
`[]string {"create -f" ,  "create ig", ...}`
and then  make(map[string]bool) ;  and fill it from range(unsupportedCommandsForVSphere  ).